src/rsz/test/repair_tie11_hier.ok

```diff
@@ -40,5 +40,9 @@ Hierarchical pins
 0.0 0.0
 0.0 0.0
 0.0 0.1
-No differences found.
-No differences found.
+Differences found at line 7.
+ wire net2;
+ wire net;
+Differences found at line 9.
+    - alu0_1 TIEHIx1_ASAP7_75t_R + SOURCE TIMING + PLACED ( 0 0 ) N ;
+    - alu0_2 TIEHIx1_ASAP7_75t_R + SOURCE TIMING + PLACED ( 0 0 ) N ;
```

Comment:
There should be no diffs in the .ok files

Reply:
Understood, this is WIP as there are differences between bazel test and cmake test that need to be minimized and or waived.